api/routes.js

```diff
@@ -444,7 +444,10 @@ var wt2html = function( req, res, wt ) {
 
 routes.interParams = function( req, res, next ) {
 	if ( req.params[0].match(/https?:\/\//) ) {
-		parsoidConfig.setInterwiki( req.params[0], req.params[0] );
+		// Route requests from staging environments to the appropriate MediaWiki deployment
+		const envMatch = /^.*\.(verify|preview|sandbox[^\.]*)\.(fandom\.com|wikia\.(com|org))/.exec(req.params[0]);
```

Comment:
is verify still a thing?

Reply:
yup, it is: https://verify.starwars.fandom.com